CONTRIBUTING.rst

```diff
@@ -56,7 +56,7 @@ on equistore:
 - **the rust compiler**: you will need both ``rustc`` (the compiler) and
   ``cargo`` (associated build tool). You can install both using `rustup`_, or
   use a version provided by your operating system. We need at least Rust version
-  1.56 to build equistore.
+  1.59 to build equistore.
 - **Python**: you can install ``Python`` and ``pip`` from your operating system.
   We require a Python version of at least 3.6.
```

Comment:
Should the "We require a Python version of at least 3.x" be updated?

Reply:
But 3.x would also mean 3.5 which we do not support.

Reply:
Apologies, perhaps I was too ambiguous with what I meant here - I wasn't suggesting a change to "3.x", just to the specific and appropriate version (i.e. "3.10"), consistent with the named versions in all other files.

Reply:
Ah I see. Thanks! Looks good to me.